src/jsx/modal.jsx

```diff
@@ -64,7 +64,7 @@ ModalMessage.propTypes = {
     message: React.PropTypes.object.isRequired,
     primaryButton: React.PropTypes.string,
     primaryButtonClicked: React.PropTypes.func.isRequired,
-    secondaryButton: React.PropTypes.func,
+    secondaryButton: React.PropTypes.string,
```

Comment:
To avoid such misconceptions in the future, you should perhaps add the `label` suffix so that it is clear what you are talking about. :smile: